designs/webviews/webviews-general.md

```diff
@@ -0,0 +1,95 @@
+# Webviews for Rich UIs in VS Code
+
+## Objective
+
+Justify expanding the criteria for using webviews within the AWS Toolkit for VS Code in order to create richer UIs than VS Code natively allows.
+
+### Out-of-scope
+
+* Architectures or frameworks for the webviews
+  * A best-practices guide to developing webviews with the architecture/framework decided on above.
+* Look and feel discussions
+* A detailed list of user experiences to port to webviews
```

Comment:
Treat documentation as timeless. This line likely wouldn't make sense to read six months from now. It may be sufficient to state "pre-existing portions of the toolkit may not be in alignment with this design"